<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>820</width>
         <height>590</height>
        </rect>
       </property>
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>190</y>
          <width>61</width>
          <height>15</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">margin: auto; 

padding: 10px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Ima&amp;ge</string>
    </property>
    <widget class="QMenu" name="menuFilters">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>&amp;Filters</string>
     </property>
     <addaction name="actionFilter_1"/>
     <addaction name="actionFilter_4"/>
     <addaction name="actionFilter_5"/>
    </widget>
    <widget class="QMenu" name="menuBrightness">
     <property name="title">
      <string>&amp;Brightness</string>
     </property>
     <addaction name="actionBrightnessMore"/>
     <addaction name="actionBrightnessLess"/>
    </widget>
    <addaction name="actionNoise"/>
    <addaction name="menuBrightness"/>
    <addaction name="menuFilters"/>
    <addaction name="separator"/>
    <addaction name="actionImage_Properties"/>
   </widget>
   <widget class="QMenu" name="menuColors">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;Colors</string>
    </property>
    <property name="toolTipsVisible">
     <bool>false</bool>
    </property>
    <addaction name="actionInvert"/>
    <addaction name="actionBlackWhite"/>
    <addaction name="actionGrayscale"/>
    <addaction name="actionSepia"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>E&amp;dit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuImage"/>
   <addaction name="menuColors"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionInvert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Invert</string>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Grayscale</string>
   </property>
  </action>
  <action name="actionMore">
   <property name="text">
    <string>&amp;More</string>
   </property>
  </action>
  <action name="actionLess">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Less</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sa&amp;ve as...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionFilter_2">
   <property name="text">
    <string>Filter 2</string>
   </property>
  </action>
  <action name="actionFilter_3">
   <property name="text">
    <string>Filter 3</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionFilter_1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Filter 1</string>
   </property>
  </action>
  <action name="actionFilter_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter &amp;2</string>
   </property>
  </action>
  <action name="actionFilter_5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter &amp;3</string>
   </property>
  </action>
  <action name="actionImage_Properties">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Image Properties</string>
   </property>
  </action>
  <action name="actionBlackWhite">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Black-White</string>
   </property>
  </action>
  <action name="actionSepia">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Sepia</string>
   </property>
  </action>
  <action name="actionNoise">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Noise</string>
   </property>
  </action>
  <action name="actionBrightnessMore">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;More</string>
   </property>
  </action>
  <action name="actionBrightnessLess">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Less</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
